src/client/Microsoft.Identity.Client/ManagedIdentity/ManagedIdentitySource.cs

```diff
@@ -53,6 +53,11 @@ public enum ManagedIdentitySource
         /// <summary>
         /// The source to acquire token for managed identity is Machine Learning Service.
         /// </summary>
-        MachineLearning
+        MachineLearning,
+
+        /// <summary>
+        /// The source to acquire token for managed identity is Service Fabric Federated.
+        /// </summary>
+        ServiceFabricFederated
```

Comment:
I would use the term `FMI` instead of `Federated`, because federated is very overloaded.